mod/cp_notifications/views/default/cp_notifications/email_template.php

```diff
@@ -358,6 +358,13 @@
 		$cp_topic_description_discussion_en = gc_explode_translation($cp_topic_description_discussion,'en');
 		$cp_topic_description_discussion_fr = gc_explode_translation($cp_topic_description_discussion,'fr');
 
+
+		if ($vars['cp_topic']->getSubtype() === 'thewire') {
+			$cp_notify_msg_title_en = elgg_echo('cp_notify:body_new_content:title3', array($vars['cp_topic']->getOwnerEntity()->getURL(), $vars['cp_topic']->getOwnerEntity()->name, cp_translate_subtype($vars['cp_topic']->getSubtype())), 'en');
+			$cp_notify_msg_title_fr = elgg_echo('cp_notify:body_new_content:title3', array($vars['cp_topic']->getOwnerEntity()->getURL(), $vars['cp_topic']->getOwnerEntity()->name, cp_translate_subtype($vars['cp_topic']->getSubtype()), false), 'en');
+		}	$wire_post_message = "<p>{$vars['cp_topic']->description}</p>";
```

Comment:
Is this supposed to be in the conditional?

Reply:
yes it is!